website_timecheck/views/templates.xml

```diff
@@ -342,18 +341,23 @@
                     >
                         <span class="fa fa-shopping-cart" /> Checkout
                     </a>
+                    <a
+                        groups="base.group_public"
+                        href="/web/login"
+                        class="btn btn-secondary btn-sm"
+                        aria-label="Shopping cart"
+                        title="Shopping cart"
+                    >
+                        <span class="fa fa-shopping-cart" /> Checkout
+                    </a>
```

Comment:
I don't have enough background knowledge to understand this, but is this function required on the assumption that https://github.com/qrtl/oaw-custom/tree/12.0/website_sale_login_required is not used?

Reply:
Yes, since the website_sale_login_required is not being used for now (it is uninstalled), but it might be used again once the business situation goes well.